### ISSUE TITLE
BUG: when `ls` on an argument that is already set.

### ISSUE THREAD

**Issue by alexsavio** [alexsavio/hansel#14]

**Comment by alexsavio** [comment]
```
>>> base_dir = op.expanduser('~/data/thomas/ica_out')
>>> icacr = Crumb(op.join(base_dir, '{exp}', '{file}'))
>>> icacr.replace(exp='kaka').ls('exp')

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-21-b69fa2352351> in <module>()
----> 1 icacr.replace(exp='kaka').ls('exp')

/home/alexandre/envs/pytre/lib/python3.5/site-packages/hansel-0.8.0-py3.5.egg/hansel/crumb.py in ls(self, arg_name, fullpath, make_crumbs, check_exists)
    609 
    610         self._check_ls_params(make_crumbs, fullpath)
--> 611         values_map = self.values_map(arg_name, check_exists=check_exists)
    612         if fullpath:
    613             paths = self.build_paths(values_map, make_crumbs=make_crumbs)

/home/alexandre/envs/pytre/lib/python3.5/site-packages/hansel-0.8.0-py3.5.egg/hansel/crumb.py in values_map(self, arg_name, check_exists)
    533             return [list(self.arg_values.items())]
    534 
--> 535         arg_deps   = self._arg_parents(arg_name)
    536         values_map = None
    537         for arg in arg_deps:

/home/alexandre/envs/pytre/lib/python3.5/site-packages/hansel-0.8.0-py3.5.egg/hansel/crumb.py in _arg_parents(self, arg_name)
    487         arg_deps: Mapping[str, int]
    488         """
--> 489         dpth, _, _ = _find_arg_depth(self.path, arg_name)
    490         return OrderedDict([(arg, idx) for idx, arg in self._open_arg_items() if idx <= dpth])
    491 

TypeError: 'NoneType' object is not iterable
```

**Comment by alexsavio** [comment]
Solved in 0.9.0.
